src/views/Give/CausesDashboard.tsx

```diff
@@ -124,18 +124,16 @@ export default function CausesDashboard() {
             >
               You can direct your yield to a recipient of your choice
             </Typography>
-            <Button
-              variant="outlined"
-              color="primary"
+            <TertiaryButton
               className="custom-give-button"
               onClick={() => handleCustomGiveButtonClick()}
               disabled={!address}
             >
               <Typography variant="body1" style={{ marginBottom: "0px" }}>
                 <Trans>Custom Recipient</Trans>
               </Typography>
-            </Button>
-          </Paper>
+            </TertiaryButton>
```

Comment:
This suggestion will look 'slightly' different than the current button. But it moves the button to be consistent with the font sizes from our V2 design system. FYI that v1.7.0 (you'd have to upgrade on this branch) of component library also has support for various font sizes based on design system revisions i.e. if the size="small" prop is is also passed, you'll get a button height that is shorter and a smaller font size. 
```suggestion
            <TertiaryButton onClick={() => handleCustomGiveButtonClick()} disabled={!address}>
              <Trans>Custom Recipient</Trans>
            </TertiaryButton>
```